components/channel_layout/center_channel/index.ts

```diff
@@ -83,5 +83,5 @@ const connector = connect(mapStateToProps, mapDispatchToProps);
 
 export type PropsFromRedux = ConnectedProps<typeof connector>;
 
-export default connector(CenterChannel);
+export default withRouter(connector(CenterChannel));
```

Comment:
to access router props

Reply:
Interesting that this seems to work fine now. Somewhere else that we were using `withRouter` recently had some TypeScript issues that were really hard to address